hydradx/model/amm/centralized_market.py

```diff
@@ -127,7 +131,7 @@ def swap(
         if (base, quote) not in self.order_book:
             if (quote, base) in self.order_book:
                 base, quote = quote, base
-            else:
+            if (base, quote) not in self.order_book:
```

Comment:
didn't this already get checked 3 lines above?

Reply:
Yes, but the two may have been reversed by this time. It's functionally the same as 'else', and I can't remember why I changed it now. I think this way is more confusing, so I'll change it back.